# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: browser/components/sessionstore/test/browser_pinned_tabs.js
# Commit: 33b38d809b51
# Full Hash: 33b38d809b51cfda54c6412513b0e67f197526b5
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2023-03-06 21:17:18
# Description:
#   Bug 1820207 - await BrowserTestUtils.crashFrame cleaning up crashes in browser_pinned_tabs.js. r=NeilDeakin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D171722
# ==============================================================================

diff -r 41f8f9a43e5e -r 33b38d809b51 browser/components/sessionstore/test/browser_pinned_tabs.js
--- a/browser/components/sessionstore/test/browser_pinned_tabs.js	Mon Mar 06 15:41:40 2023 +0000
+++ b/browser/components/sessionstore/test/browser_pinned_tabs.js	Mon Mar 06 16:02:17 2023 +0000
@@ -306,7 +306,7 @@
   await allTabsRestored;
 
   let tabs = gBrowser.tabs;
-  BrowserTestUtils.crashFrame(tabs[17].linkedBrowser);
+  await BrowserTestUtils.crashFrame(tabs[17].linkedBrowser);
 
   await TestUtils.topicObserved("sessionstore-state-write-complete");
 
